### ISSUE TITLE
EMASPREAD Pairs Log not showing correct values

### ISSUE THREAD

**Issue by mightysoul** [taniman/profit-trailer#1780]
EMASPREAD values are correct on **Possible Buy Log** but as soon as the coin is bought the values are incorrect on **Pairs log**. None of the values are match with TV for pairs log. 

Here is an example;

WPR should be negative spread but PT shows its positive!.

![image](https://user-images.githubusercontent.com/37293938/39612966-a5b63906-4f6c-11e8-88d2-2c7ecb5758cd.png)

![image](https://user-images.githubusercontent.com/37293938/39612956-8f80df4c-4f6c-11e8-8895-2b6fa31c1f65.png)

**Comment by mightysoul** [comment]
Why this case is closed without any explanation. It is a paid product with license and is this how you treat to your customers?

**Comment by SgtBatten** [comment]
Possibly because it's patched in an upcoming release. Seeing as it was assigned to someone and then closed by them

**Comment by mightysoul** [comment]
I hope it is fixed but we as customers won't be able to understand if it is fixed. It isn't that hard to comment "it is fixed for upcoming release". Anyway thanks for the update SgtBatten.

**Comment by helo03** [comment]
The issue is fixed in an upcoming release.

mightysoul <notifications@github.com> schrieb am So., 13. Mai 2018, 07:50:

**Comment by helo03** [comment]
Again me. Previous comment has been sent via mobile. So to go a little deeper and explain more detailed.
You as customers have the right to know if something is fixed or not - i agree on that. We as the ProfitTrailer team though, will not disregard any bugs. So if you have an issue, you open it and if it's closed this means we can't reproduce it (in that case we will ask for more details though) or we are going to work on fixing it. (someone from the development team has it on his list).

I understand your request for more information on closed issues and as this has been assigned to me and closed by me I will try to give more details next time.

Long story short: Should be fixed in upcoming release. EMASPREAD has not been working correctly as EMASPREAD has been using EMAGAIN values. If you still encounter it after the next release, please write answer to this issue again.
Kind Regards,
Mario
